show-line-of-sight-between-geoelements/README.md

```diff
@@ -0,0 +1,41 @@
+# Show line of sight between geoelements
+
+Show a line of sight between two moving objects.
+
+![Image of Show Line of Sight Between Geoelements](show-line-of-sight-between-geoelements.png)
+
+## Use case
+
+A line of sight between geoelements (i.e. observer and target) will not remain constant whilst one or both are on the move.
+
+A line of sight is therefore useful in cases where visibility between two geoelements requires monitoring over a period of time in a partially obstructed field of view (such as buildings in a city).
+
+## How to use the sample
+
+A line of sight will display between a point on the Empire State Building (observer) and a taxi (target). The taxi will drive around a block and the line of sight should automatically update. The taxi will be highlighted and blinking when it is visible. A red segment on the line means the view between observer and target is obstructed, whereas cyan means the view is unobstructed. You can change the observer height with the slider to see how it affects the target's visibility.
+
+## How it works
+
+1. Instantiate an `AnalysisOverlay` and add it to the `SceneView`'s analysis overlays collection.
+2. Instantiate a `GeoElementLineOfSight`, passing in observer and target `GeoElement`s (features or graphics). Add the line of sight to the analysis overlay's analyses collection.
+3. To get the target visibility when it changes, react to the target visibility changing on the `GeoElementLineOfSight` instance.
+
+## Relevant API
+
+* AnalysisOverlay
+* GeoElementLineOfSight
+* LineOfSight.TargetVisibility
+
+## Offline data
+
+| Link                                                                                  | Local Location                                             |
+|---------------------------------------------------------------------------------------|------------------------------------------------------------|
+| [Taxi CAD](https://www.arcgis.com/home/item.html?id=3af5cfec0fd24dac8d88aea679027cb9) | `<userhome>/ArcGIS/Runtime/Data/3D/dolmus\_3ds/dolmus.zip` |
```

Comment:
We do not use the "Offline data" section anymore since the sample will now provision the download itself, unlike previously with runtime it was a manual ADB load on the device. 

```suggestion
```